### PR TITLE
intial scaffold

### PR DESCRIPTION
**Pull request by slugbyte**
added empty index.html style.css app.js

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
```
Comment (slugbyte) [inline src/index.html:2]: I approve this

**Review by slugbyte — COMMENTED** [review]
keep up the good work 
// notes

**Review by cr-gonzalez — COMMENTED** [review]
:)

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  <head>
+  </head>
+  <body>
+    <h1> lulwat </h1>
```
Comment (cr-gonzalez) [inline src/index.html:6]: :)

**Review by cr-gonzalez — COMMENTED** [review]
Does it work now?

**Pull request merged**